packages/enzyme-adapter-react-16/src/ReactSixteenAdapter.js

```diff
@@ -113,11 +113,21 @@ function toTree(vnode) {
     }
     case HostText: // 6
       return node.memoizedProps;
+    case ForwardRef: {
```

Comment:
This is the crux of the change. I realise that this `toTree` function is largely adapted from `toTree` in `react-test-renderer`. This creates a divergence in behaviour between the two libraries which sounds scary. Ideally, this change would go into `react-test-renderer` and then enzyme could use the toTree function directly.